client/lib/discounts/active-discounts.js

```diff
@@ -54,4 +55,22 @@ export default [
 		plansPageNoticeText:
 			'Improve your SEO, branding, credibility, and even word-of-mouth marketing with a custom domain. All plan upgrades include a free domain name of your choice.',
 	},
+	{
+		name: 'august20',
+		startsAt: new Date( 2018, 7, 6, 0, 0, 0 ),
+		endsAt: new Date( 2018, 7, 11, 0, 0, 0 ),
+		nudgeText: translate( '20% Off All Plans' ),
+		ctaText: translate( 'UPGRADE' ),
+		plansPageNoticeText: translate(
+			'Enter coupon code “AUGUST20” during checkout to claim your 20% discount.'
```

Comment:
Here, a variable for the coupon code would have been better, for one to ensure no accidential translation, and secondly we might re-use this with another coupon code later on.